### PR TITLE
Progress Bar Support

### PR DESCRIPTION
**Pull request by dpetrovics**
#46 and stacked progress bars.

### CODE REVIEW

File: src/om_bootstrap/progress_bar.cljs [hunk 1]
```diff
@@ -3,61 +3,65 @@
...
-    :striped? s/Bool
-    :active? s/Bool}))
+   {:now s/Int
+    (s/optional-key :min) s/Int
+    (s/optional-key :max) s/Int
+    (s/optional-key :label) t/Renderable
+    (s/optional-key :sr-only?) (s/named s/Bool "Screenreader-only? Hide the label?")
+    (s/optional-key :striped?) s/Bool
+    (s/optional-key :active?) s/Bool
+    (s/optional-key :nested?) (s/named s/Bool "Specify this for a nested ProgressBar inside a stacked ProgerssBar.")}))
```
Comment (dpetrovics) [inline src/om_bootstrap/progress_bar.cljs]: I added this key - false by default - to handle the case of stacked progress bars. The react-bootstrap guys use keys (https://github.com/react-bootstrap/react-bootstrap/blob/master/docs/examples/ProgressBarStacked.js), but I thought the :nested? boolean might be more appropriate for us

Comment (sritchie) [inline src/om_bootstrap/progress_bar.cljs]: okay - this is another good spot for clone-with-props.

Comment (sritchie) [inline src/om_bootstrap/progress_bar.cljs]: typo on ProgressBar

File: src/om_bootstrap/progress_bar.cljs [hunk 2]
```diff
@@ -3,61 +3,65 @@
   work."
   (:require [om-bootstrap.types :as t]
             [om-tools.dom :as d :include-macros true]
-            [schema.core :as s])
-  (:require-macros [schema.macros :as sm]))
+            [schema.core :as s :include-macros true]
```
Comment (sritchie) [inline src/om_bootstrap/progress_bar.cljs]: nope, this doesn't work with the version of schema in this library. We've gotta keep it the old way until we force users to upgrade to Schema 0.3.x.

File: src/om_bootstrap/progress_bar.cljs [hunk 3]
```diff
@@ -3,61 +3,65 @@
...
       (* 100)
       (Math/ceil)))
 
-(sm/defn sr-only-label :- t/Component
-  "Renders a screenreader-only label."
-  [label :- t/Renderable]
-  (d/span {:class "sr-only"} label))
-
-(comment
-  (sm/defn render-label
```
Comment (sritchie) [inline src/om_bootstrap/progress_bar.cljs]: can't remember what this was.

**Comment by sritchie** [comment]
Anything we're missing from the Bootstrap funcs?

File: src/om_bootstrap/progress_bar.cljs (hunk 2 above)
Comment (dpetrovics) [inline src/om_bootstrap/progress_bar.cljs]: reverting

File: src/om_bootstrap/progress_bar.cljs (hunk 1 above)
Comment (dpetrovics) [inline src/om_bootstrap/progress_bar.cljs]: fixed

**Comment by dpetrovics** [comment]
OK this looks solid, thanks @dignati ! Merging into develop

**Pull request merged**